### ISSUE TITLE
Failed to compile

### ISSUE THREAD

**Issue by cbroberg** [styled-components/colors-demo#1]
Failed to compile.

Error in ./src/Tiles/index.js
Module not found: 'polished/lib/internalHelpers/_parseColorString' in /Users/cbroberg/Apps/apps-play/color-schemer/src/Tiles

 @ ./src/Tiles/index.js 13:24-81

**Comment by truman303** [comment]
Yip, tried updating polished to no avail.

Arrived here through Josh Morony post on styling an ionic app.

**Comment by mxstbr** [comment]
Yeah the imports need to be fixed for the new version of polished, mind submitting a PR?
